ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue

```diff
@@ -34,16 +86,55 @@
 <script lang="ts">
 import { defineComponent, reactive, toRefs } from '@vue/composition-api'
 import { RouteNames } from '@/enums'
+import { useGetters } from 'vuex-composition-helpers'
+import { useCountriesProvinces, formatAddress } from '@/composables/address/factories'
 
 export default defineComponent({
   name: 'SubmittingPartyReview',
   components: {},
   props: {},
   setup () {
+    const {
+      getMhrRegistrationSubmittingParty,
+      getMhrAttentionReferenceNum
+    } = useGetters<any>([
+      'getMhrRegistrationSubmittingParty',
+      'getMhrAttentionReferenceNum'
+    ])
+
     const localState = reactive({})
 
+    const parseAddress = () => {
```

Comment:
We can use a `BaseAddress` component here which can display the address with all the fields.
E.g. `<base-address :schema="addressSchema" :value="address" />`

Reply:
updated, now using the `baseAddress` component.

Reply:
Great, thanks!